From 01560cac5ddbbb93d642c29758c27909ce1c1d18 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Wed, 13 Nov 2019 15:24:27 -0500
Subject: [PATCH 38/45] Rename our installed .o and linker scripts.

Since crt0 and our linker scripts are now in a subdirectory, there's no
need to make them named differently in the installed machine.  This
renames them to crt0.o and efi.lds .

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 gnuefi/Makefile | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/gnuefi/Makefile b/gnuefi/Makefile
index 5bdaec7579b..d96ec74e96f 100644
--- a/gnuefi/Makefile
+++ b/gnuefi/Makefile
@@ -55,21 +55,21 @@ all:	$(TARGETS)
 libgnuefi.a: $(OBJS)
 	$(AR) $(ARFLAGS) $@ $^
 
-
 clean:
 	@rm -vf $(TARGETS) *~ *.o $(OBJS)
 
 install:
-	mkdir -p $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)
-	$(INSTALL) -m 644 -t $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)/ $(TARGETS)
+	@mkdir -p $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)
+	@$(INSTALL) -v -m 644 -t $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH) libgnuefi.a
+	@$(INSTALL) -v -m 644 crt0-efi-$(ARCH).o $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)/crt0.o
 ifneq (,$(findstring FreeBSD,$(OS)))
  ifeq ($(ARCH),x64)
-	$(INSTALL) -m 644 $(SRCDIR)/elf_$(ARCH)_fbsd_efi.lds $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)
+	@$(INSTALL) -v -m 644 $(SRCDIR)/elf_$(ARCH)_fbsd_efi.lds $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)
  else
-	$(INSTALL) -m 644 $(SRCDIR)/elf_$(ARCH)_efi.lds $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)
+	@$(INSTALL) -v -m 644 $(SRCDIR)/elf_$(ARCH)_efi.lds $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)/efi.lds
  endif
 else
-	$(INSTALL) -m 644 $(SRCDIR)/elf_$(ARCH)_efi.lds $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)
+	@$(INSTALL) -v -m 644 $(SRCDIR)/elf_$(ARCH)_efi.lds $(INSTALLROOT)$(LIBDIR)/gnuefi/$(ARCH)/efi.lds
 endif
 
 include $(SRCDIR)/../Make.rules
-- 
2.24.1

